wp-graphql.php

```diff
@@ -684,6 +690,45 @@ public static function get_app_context() {
 
 			return $app_context;
 		}
+
+		/**
+		 * Adds default GraphQL post-types settings
+		 *
+		 * @param WP_Post_Type|array $args       Post type arguments.
+		 * @param string             $post_type  Post type
+		 * 
+		 * @return array
+		 */
+		public static function add_post_type_settings( $args, $post_type ) {
+			global $wp_post_types;
+
+			$post_type_settings = [
+				'show_in_graphql'                 => true,
+				'graphql_single_name'             => str_replace( '_', '', ucwords( strtolower( $post_type ), '_' ) ),
+				'graphql_plural_name'             => str_replace( '_', '', ucwords( strtolower( $post_type ), '_' ) ) . 's',
+				'register_schema_type'            => true,
+				'register_root_query'             => true,
+				'register_root_connection'        => true,
+				'connect_to_comments'             => true,
+				'connect_to_taxonomies'           => true,
+				'supports_content_node'           => true,
+				'supports_post_object_union'      => true,
+				'supports_menu_item_object_union' => true,
+				'resolve_from_uri'                => true,
+				'use_post_object_loader'          => true,
```

Comment:
I wonder if instead of a true/false here it should just be the name of the loader to use?

Reply:
Good idea :+1: I'll see what I can wire up.